code/modules/reagents/dispenser/dispenser_presets.dm

```diff
@@ -27,33 +27,21 @@
 	name = "medicine dispenser"
 	spawn_cartridges = list(
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/inaprov,
-			/obj/item/weapon/reagent_containers/chem_disp_cartridge/ryetalyn,

```

Comment:
The PR description sais that you are removing "hyperzine" from the chem dispenser.
You are removing much more than just hyperzine from here.